src/tls/cockpit-certificate-helper.in

```diff
@@ -10,15 +10,15 @@ COCKPIT_RUNTIME_DIR="/run/cockpit"
 
 install_cert() {
     local destination="${COCKPIT_WS_CERTS_D}/$1"
-    mv -Z "$1" "${destination}"
+    mv -Z "$1" "${destination}" || mv "$1" "${destination}"
```

Comment:
This hides errors with the first `mv` call, and also leaves unsightly/confusing error messages in the journal.

Perhaps something like `if  [ -d /sys/fs/selinux ]; then MV_OPTS="-Z"; fi` at the beginning of the script?

Reply:
:+1: Good idea !